packages/@react-spectrum/overlays/stories/Modal.stories.tsx

```diff
@@ -58,24 +60,30 @@ function UnmountingTrigger() {
     setModalOpen(true);
   };
 
+  let closeModal = () => {
```

Comment:
is this just a story workaround for the PR we were just looking at right? do we need to remember to remove it later?

Reply:
This should eventually be resolved by whatever fix we devise for https://github.com/adobe/react-spectrum/pull/2445, but it illustrates the consideration one must have for managing focus when a dialog closes.